src/store/zns/index.ts

```diff
@@ -19,8 +18,9 @@ export interface ZnsState {
 
 const initialState: ZnsState = {
   value: {
-    route: config.defaultZnsRoute,
-    deepestVisitedRoute: config.defaultZnsRoute,
```

Comment:
Do we still need `defaultZnsRoute` in the config file or can we remove it entirely?

Reply:
it's still used. just not in this spot. this default actually never even ended up getting used anyway.